packages/patoline/patoline.0.1/opam

```diff
@@ -27,6 +27,7 @@ depends: [
 ]
 install: [make "install"]
 synopsis: "A new typesetting system, programmable in ocaml."
+dev-repo: "git+https://github.com/patoline/patoline.git"
 url {
   src: "http://patoline.org/darcs/patoline/patoline-0.1.tar.gz"
```

Comment:
```suggestion
  src: "https://opam.ocaml.org/cache/md5/2f/2f7fe33ef6dff305e9a2407696c7a799"
```

The website no-longer exists and the release is not tagged. I am going to point this to the cache. Could be nice to release their last 0.2 release